CPAC/pipeline/cpac_pipeline.py

```diff
@@ -2378,91 +2489,119 @@ def build_workflow(subject_id, sub_dict, c, pipeline_name=None, num_ants_cores=1
                     }, override=True)
 
                     new_strat.append_name(regressor_workflow.name)
-                    new_strat.append_name(nuisance_regression_before_workflow.name)
-                
+                    new_strat.append_name(
+                        nuisance_regression_before_workflow.name)
+
                     if 'Bandpass' in regressors_selector:
                         new_strat.append_name(filtering.name)
 
                 new_strat_list.append(new_strat)
 
-        # Be aware that this line is supposed to override the current strat_list: it is not a typo/mistake!
-        # Each regressor forks the strategy, instead of reusing it, to keep the code simple
+        # Be aware that this line is supposed to override the current
+        # strat_list: it is not a typo/mistake!
+        # Each regressor forks the strategy, instead of reusing it, to keep
+        # the code simple
         strat_list = new_strat_list
 
-
         # Inserting Median Angle Correction Workflow
         new_strat_list = []
 
-        # TODO ASH normalize w schema val
-        if 1 in c.runMedianAngleCorrection:
+        if '3-median-angle-correction' in c.nuisance_corrections:
```

Comment:
I made median angle correction an optional key (also in the schema https://github.com/FCP-INDI/C-PAC/blob/6d8f0269b9821ad2ccd634a2719013ae70775ae9/CPAC/pipeline/schema.py#L226-L229 though if we have `'3-median-angle-correction'` it needs to be a dict with `run` and `target_angle_deg` keys), so it's just skipped if not included.

Reply:
I'm not totally sold in the `3-` in an optional key though.

Reply:
Resolved in 14db696